BlogTemplate/wwwroot/css/site.css

```diff
@@ -10,74 +14,159 @@
 }
 
 a {
-    color: dodgerblue;
+    color: var(--link-color);
 }
 
 nav {
-    background-color: #555;
-    padding: 10px 0;
+    background-color: var(--nav-color);
+    height: 60px;
+    line-height: 60px;
 }
 
     nav ul {
         list-style: none;
         padding: 0;
         margin: 0;
+        font-family: Futura, Oswald, "Trebuchet MS", Arial, sans-serif;
     }
 
-    nav li:first-child {
-        font-size: 200%;
-        margin: 0;
-    }
 
-    nav li:nth-child(5) {
-        margin-left: 880px;
-        display: inline-block !important;
-    }
     nav li {
-        display: inline-block;
+        float: left;
         margin-left: 20px;
+        margin-top: 5px;
     }
 
+        nav li:first-child {
+            font-size: 200%;
+            margin: 0;
+        }
+
     nav a {
-        color: white;
+        color: var(--content-color);
         text-decoration: none;
     }
 
         nav a:hover {
-            color: #b4b4b4;
+            color: var(--hover-color);
         }
 
-article h2 a {
-    color: black;
-    text-decoration: none;
+h2 {
+    font-family: Futura, Oswald, "Trebuchet MS", Arial, sans-serif;
 }
 
-    article h2 a:hover {
-        text-decoration: underline;
+h4 {
+    font-family: Futura, Oswald, "Trebuchet MS", Arial, sans-serif;
+    line-height: 0;
+}
+
+article {
+    border: 1px solid var(--line-color);
+    padding: 25px;
+    margin: 25px 0;
+}
+
+    article h1 {
+        font-family: Futura, Oswald, "Trebuchet MS", Arial, sans-serif;
+    }
+
+    article h2 {
+        font-family: Futura, Oswald, "Trebuchet MS", Arial, sans-serif;
+    }
+
+        article h2 a {
+            color: black;
+            text-decoration: none;
+        }
+
+            article h2 a:hover {
+                text-decoration: underline;
+            }
+
+    article h4 {
+        font-family: Futura, Oswald, "Trebuchet MS", Arial, sans-serif;
+        line-height: 0;
     }
 
 footer {
-    margin-top: 50px !important;
-    border-top: 1px solid silver;
+    margin-top: 80px !important;
+    border-top: 1px solid var(--line-color);
 }
 
+    footer p {
+        color: var(--light-color);
+    }
+
+        footer p a {
+            float: right;
+        }
+
+        footer p a {
+            content: "🔒";
+        }
+
 #date {
     font-size: 12px;
-    color: darkgrey;
+    font-family: Futura, Oswald, "Trebuchet MS", Arial, sans-serif;
+    color: var(--light-color);
 }
-#Comment-Date{
-    color: darkgrey;
-    font: 12px/.5 arial;
+
+#Comment-Date {
```

Comment:
These should probably be classes (.Comment-Date) instead of IDs (#Comment-Date).

Also, CSS naming conventions use lower case, so we should be consistent with the common convention.